modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java

```diff
@@ -1673,6 +1674,7 @@ public void testErrIllegalStateWithVerbose() {
      * without {@link CommonArgParser#CMD_VERBOSE} flag.
      */
     @Test
+    @WithSystemProperty(key = IGNITE_CLUSTER_NAME, value = "TEST_CLUSTER_NAME")
```

Comment:
This particular test checks very specific case so I don't think it's a good idea to add extra checks here. Better to choose 3-5 most popular commands and add your checks to some common cases. Also best way to check control script output is to use testOut.toString().contains(%your_pattern%)